### DISCUSSION TITLE
Have visualizer on the osu! logo be more reactive when not in Kiai time

### DISCUSSION THREAD

**Discussion by ThatStella7922** [ppy/osu#13056] [Ideas]
**Describe the new feature:** 
Have the osu! logo visualizer be more reactive when the current song isn’t in Kiai time.

**Proposal designs of the feature:** 
This could possibly be a toggle in the settings or have the entire reactivity of the visualizer be adjustable by the user in the settings.

by “reactive” I mean more intensity to how high the visualizer bars go
this is a low priority feature though

**Comment by luaneko** [comment]
What exactly is meant by 'reactive'?

**Comment by ThatStella7922** [comment]
Knew I didn’t word this the best, I mean like just larger bars to make it look more “intense”. And by larger I don’t mean thicker bars, I mean like they go higher

**Comment by peppy** [comment]
I think it’s fine as is? Maybe we can offer an intensity control as a skinning option in the future.

**Comment by ThatStella7922** [comment]
Yea that’s exactly what I meant! I would love an intensity control in the future.
